packages/react/src/utils/json/parse-partial-json.ts

```
@@ -1,14 +1,59 @@
+
+const getFieldStatus = (
+  lastState: ContentPartStatus,
+  args: unknown,
+  fieldPath: string[],
+  partialCount: number,
+): ContentPartStatus => {
+  if (fieldPath.length === 0) return lastState;
+  if (partialCount === 1) return { type: "complete" };
+  if (typeof args !== "object" || args === null) return { type: "complete" };
+
+  const path = fieldPath.at(-1)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)